### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Review my code for the typography component

PREVIEW: https://vercel-ui-preview.netlify.app/component-lib/typography.html

### CODE REVIEW

**Comment by anasm11** [comment]
better to merge into a development branch first before going to main. rest all looks good.

File: component-lib/typography.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+		<link rel="stylesheet" href="./all-comp.css" />
+		<link
+			href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"
+			rel="stylesheet"
+		/>
+		<script>
+			var themeDefault = window.localStorage.getItem("data-theme");
+			if (themeDefault)
+				document.documentElement.setAttribute("data-theme", themeDefault);
+			console.log(themeDefault);
```
Comment (amarphule) [inline component-lib/typography.html]: remove console log

File: component-lib/typography.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+			const changeThemeToDark = () => {
+				document.documentElement.setAttribute("data-theme", "dark"); // set theme to dark
+				localStorage.setItem("data-theme", "dark"); // save theme to local storage
+			};
+
+			const changeThemeToLight = () => {
+				document.documentElement.setAttribute("data-theme", "light"); // set theme light
+				localStorage.setItem("data-theme", "light"); // save theme to local storage
+			};
+
```
Comment (amarphule) [inline component-lib/typography.html]: I think writing comments on each line is a bad practice. Instead, you can write a single line comment above `changeTheme()` saying "Theme change Dark-Light" something like that.

File: component-lib/typography.html [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+                            </iframe>
+						</div>
+					</div>
+					<div class="component-example">
+						<div class="title">Special Typography</div>
+						<div class="sub-title">
+							Use respective classes to acheive special Typography such highlighted text, text with line through, label text, etc.
+						</div>
+
+						<div class="component-display-container continer-vertical">
```
Comment (amarphule) [inline component-lib/typography.html:203]: please check `continer-vertical` spelling it will cause error in future

**Review by amarphule — CHANGES_REQUESTED** [review]
Your code looks good. There are some minor changes. I did reviews on that please check

File: component-lib/typography.html (hunk 3 above)
Comment (nikhiljugale007) [reply component-lib/typography.html:203]: I know it. But if I change it now I have to change at lots of places. So as soon I complete my component library I will fix that spelling.

File: component-lib/typography.html (hunk 2 above)
Comment (nikhiljugale007) [reply component-lib/typography.html]: Sure I will change this.

File: component-lib/typography.html (hunk 1 above)
Comment (nikhiljugale007) [reply component-lib/typography.html]: I will remove it

**Pull request merged**